install/lib/class.FroxlorInstall.php

```diff
@@ -1273,6 +1298,48 @@ private function _guessWebserver()
 		}
 	}
 
+
+	/**
+	 * get/guess linux distribution
+	 */
+	private function _guessDistribution()
+	{
+		// post
+		if (! empty($_POST['distribution'])) {
+			$this->_data['distribution'] = $_POST['distribution'];
+		} else {
+			$os_version = parse_ini_file('/etc/os-release', false);
+			if ($os_version['ID'] == 'debian') {
+
+			} elseif ($os_version['ID'] == 'debian') {
+                                if(explode('.',$os_version['VERSION_ID'])[0] == '8') {
```

Comment:
why explode() everytime in the if-elseif-statements if you can get the version using `$version = explode('.',$os_version['VERSION_ID'])[0];` prior of that and then always using $version

Reply:
sure